README.md

```diff
@@ -1,106 +1,120 @@
-#### CFPB Open Source Project Template Instructions
+# Wagtail-TreeModelAdmin
 
-1. Create a new project.
-2. [Copy these files into the new project](#installation)
-3. Update the README, replacing the contents below as prescribed.
-4. Add any libraries, assets, or hard dependencies whose source code will be included
-   in the project's repository to the _Exceptions_ section in the [TERMS](TERMS.md).
-  - If no exceptions are needed, remove that section from TERMS.
-5. If working with an existing code base, answer the questions on the [open source checklist](opensource-checklist.md)
-6. Delete these instructions and everything up to the _Project Title_ from the README.
-7. Write some great software and tell people about it.
+[![Build Status](https://travis-ci.org/cfpb/wagtail-treemodeladmin.svg?branch=master)](https://travis-ci.org/cfpb/wagtail-treemodeladmin)
+[![Coverage Status](https://coveralls.io/repos/github/cfpb/wagtail-treemodeladmin/badge.svg?branch=master)](https://coveralls.io/github/cfpb/wagtail-treemodeladmin?branch=master)
 
-> Keep the README fresh! It's the first thing people see and will make the initial impression.
+Wagtail-TreeModelAdmin is an extension for Wagtail's ModelAdmin that allows for a page explorer-like navigation of Django model relationships within the Wagtail admin.
+
+- [Dependencies](#dependencies)
+- [Installation](#installation)
+- [Concepts](#concepts)
+- [Usage](#usage)
+    - [Quickstart](#quickstart)
+- [API](#api)
+- [Getting help](#getting-help)
+- [Getting involved](#getting-involved)
+- [Licensing](#licensing)
+- [Credits and references](#credits-and-references)
+
+## Dependencies
+
+- Django 1.8+ (including Django 2.0)
+- Wagtail 1.13+ (including Wagtail 2.0)
+- Python 2.7+, 3.6+
 
 ## Installation
 
-To install all of the template files, run the following script from the root of your project's directory:
+1. Install wagtail-treemodeladmin:
 
-```
-bash -c "$(curl -s https://raw.githubusercontent.com/CFPB/development/master/open-source-template.sh)"
+```shell
+pip install wagtail-treemodeladmin
 ```
 
-----
+2. Add `treemodeladmin` as an installed app in your Django `settings.py`:
```

Comment:
On wagtail-sharing I added [explicit documentation](https://github.com/cfpb/wagtail-sharing/commit/5b7e912398ee6767b51a2b0bea1808077f6469b9) about its dependency on Wagtail ModelAdmin, as ModelAdmin is optional and [isn't part of the standard Wagtail setup](http://docs.wagtail.io/en/v2.0/getting_started/integrating_into_django.html).